construction_progress_management/data/base_automation.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="automation_update_progress_increment_on_change" model="base.automation">
+        <field name="name">Construction - Update Sale Order Line Progress Increment</field>
+        <field name="model_id" ref="sale.model_sale_order_line"/>
+        <field name="action_server_ids" eval="[(6, 0, [ref('construction_progress_management.action_sol_update_progress_increment_on_change')])]"/>
+        <field name="trigger">on_change</field>
+        <field name="on_change_field_ids" eval="[(6, 0, [ref('construction_progress_management.field_sale_order_line_quantity_increment')])]"/>
+    </record>
+    <record id="automation_update_quantity_increment_on_change" model="base.automation">
+        <field name="name">Construction - Update Sale Order Line Progress Increment</field>
+        <field name="model_id" ref="sale.model_sale_order_line"/>
+        <field name="action_server_ids" eval="[(6, 0, [ref('construction_progress_management.action_sol_update_quantity_increment_on_change')])]"/>
+        <field name="trigger">on_create_or_write</field>
```

Comment:
I don't get why one is on_create_or_write and the other (upper in the file) is on_change